include/openthread/instance.h

```diff
@@ -53,7 +53,7 @@ extern "C" {
  * @note This number versions both OpenThread platform and user APIs.
  *
  */
-#define OPENTHREAD_API_VERSION (231)
+#define OPENTHREAD_API_VERSION (232)
```

Comment:
Will need to rebase and increment this value again.

Reply:
Thanks - done...